### PR TITLE
Feature/110 configure environment

### PR DESCRIPTION
**Pull request by IrvingJVR-MX**
Added 2 environment (development/production)

### CODE REVIEW

File: RoutesApp_ios/utilities/helpers/ConstantVariables.swift [hunk 1]
```diff
@@ -100,5 +104,13 @@ class ConstantVariables {
 
     // Transformer Name
     static let transformerName = "CoordinateTransformer"
+     
+    // Google Maps keys
+    static let gmsServicesProvideAPIKeyProduction = ""
+    static let gmsPlacesClientProvideAPIKeyProduction  = ""
+    static let gmsServicesProvideAPIKeyDevelopment = ""
+    static let gmsPlacesClientProvideAPIKeyDevelopment = ""
```
Comment (YawarValeriano) [inline RoutesApp_ios/utilities/helpers/ConstantVariables.swift]: Is it possible to call the static variables from Env.swift?
To set the keys here

**Review by YawarValeriano — CHANGES_REQUESTED** [review]

File: RoutesApp_ios/utilities/helpers/ConstantVariables.swift (hunk 1 above)
Comment (IrvingJVR-MX) [reply RoutesApp_ios/utilities/helpers/ConstantVariables.swift]: ammm i don´t understand, it would be better to change them to env?

Comment (jmacboy) [reply RoutesApp_ios/utilities/helpers/ConstantVariables.swift]: they already have an Env.swift file on the project

**Review by jmacboy — APPROVED** [review]
approved from my side, just check Yawar's comment

File: RoutesApp_ios/utilities/helpers/ConstantVariables.swift (hunk 1 above)
Comment (IrvingJVR-MX) [reply RoutesApp_ios/utilities/helpers/ConstantVariables.swift]: done:)

**Review by YawarValeriano — APPROVED** [review]

**Pull request merged**